webpack.common.config.ts

```diff
@@ -187,6 +189,23 @@ module.exports = {
       minify: htmlMinifyConfig,
       inject: false
     }),
+    new HtmlWebpackPlugin({
+      chunks: ['blog_admin'],
+      filename: 'blog-admin-page.mainpage.html',
+      hybrid: false,
```

Comment:
hybrid field is not required. It can be removed.

Reply:
This should be `true'.

Reply:
Done.